packages/forms/src/fields/ArrayField.js

```diff
@@ -199,6 +198,11 @@ class ArrayField extends Component {
 		if (isFixedItems(schema) && allowAdditionalItems(schema)) {
 			itemSchema = schema.additionalItems;
 		}
+		formData.map((item) => {
+			const i = item || {};
+			i.isClosed = true;
```

Comment:
create a const just to assign a value if exists ?
i && i.isClosed = true 
_.set(i, 'isClosed', true)